boards/remote-revb/include/board.h

```diff
@@ -110,6 +110,19 @@
 #define CC1200_GPD2_GPIO    GPIO_PB0
 /** @} */
 
+/**
+ * @name Onboard micro-sd slot pin definitions
+ * @{
+ */
+#define SDCARD_SPI_PARAM_SPI       SPI_1
```

Comment:
ups, never mind, `SPI_1` is just fine here, the SPI_DEV() define is not yet merged...